packages/core/src/components/button/button.story.tsx

```diff
@@ -6,7 +6,12 @@ export interface ButtonProps extends BaseProps {
   href?: string;
 }
 
-export const Button = ({ href, kind, variant, ...props }: ButtonProps) =>
+export const Button = ({
+  href,
+  kind = 'action',
+  variant = 'primary',
```

Comment:
This was actually done on purpose, because for CSS there is no defaults, you *must* set every modifier manually.

And I don't even think this should belong on this PR?